src/user-guide/installing-workflows.rst

```diff
@@ -235,10 +235,21 @@ Looking at the directory structure that has been created
 Upon running ``cylc install``, symlinks for the directories ``run``, ``log``,
 ``share``, ``share/cycle`` and ``work`` will be created in accordance with
 the symlink rules for ``localhost`` as defined in
-:cylc:conf:`global.cylc[symlink dirs]`.
+:cylc:conf:`global.cylc[install][symlink dirs]`.
 
-This is overridable via the command line option ``--no-symlinks``, where the
-directories will not be symlinked.
+This is overridable via the command line option ``--symlink-dirs="log=$DIR,
+run=/path/to/dir,..."``, where the directories supplied will be used to create
+symlinks, rather than the ones specified in
+:cylc:conf:`global.cylc[install][symlink dirs]`.
```

Comment:
IMO 
- Better to explicitly use a noun rather than "this".
- "overrideable" - possibly not the easiest construction to read. 

```suggestion
You can override the default locations set in :cylc:conf:`global.cylc[install][symlink dirs]` using the command
line option ``--symlink-dirs="log=$DIR, run=/path/to/dir,..."``.
```